src/components/Order_page/Order_page.scss

```diff
@@ -0,0 +1,18 @@
+.grid {
+  display: grid;
+  grid-template-columns: 1fr;
+  row-gap: 15px;
+  column-gap: 15px;
+}
+
+@media (min-width: 750px) {
+  .grid {
+    grid-template-columns: repeat(2, 1fr);
+  }
+}
+
+@media (min-width: 1000px) {
+  .grid {
+    grid-template-columns: repeat(6, 1fr);
+  }
+}
```

Comment:
This SCSS can be made dryer by nesting the media queries in the original grid selector, like this

```
.grid {
     foo: bar;
     @media (min-width: 1000px) {
            bar: baz;
     }
}
```

This just means you don't have to repeat the .grid selector